requirements/default.txt

```diff
@@ -51,12 +53,14 @@ six==1.12.0 \
     --hash=sha256:3350809f0555b11f552448330d0b52d5f24c91a322ea4a15ef22629740f3761c \
     --hash=sha256:d16a0141ec1a18405cd4ce8b4613101da75da0e9a7aec5bdd4fa804d0e0eba73
 
+# NOTE(willkg): In order to update these, we have to redo how the s3mock works
+# in the tests.
 boto3==1.7.84 \
     --hash=sha256:0ed4b107c3b4550547aaec3c9bb17df068ff92d1f6f4781205800e2cb8a66de5 \
-    --hash=sha256:64496f2c814e454e26c024df86bd08fb4643770d0e2b7a8fd70055fc6683eb9d
+    --hash=sha256:64496f2c814e454e26c024df86bd08fb4643770d0e2b7a8fd70055fc6683eb9d # pyup:ignore <1.8.0
 botocore==1.10.84 \
     --hash=sha256:380852e1adb9ba4ba9ff096af61f88a6888197b86e580e1bd786f04ebe6f9c0c \
-    --hash=sha256:d3e4b5a2c903ea30d19d41ea2f65d0e51dce54f4f4c4dfd6ecd7b04f240844a8
+    --hash=sha256:d3e4b5a2c903ea30d19d41ea2f65d0e51dce54f4f4c4dfd6ecd7b04f240844a8 # pyup:ignore <1.11.0
```

Comment:
We have a home-grown s3mock that we use. That mocks out a certain library that botocore < 1.12.0 uses. 1.12.0 and later switch how they send HTTP requests, so we need to rewrite the s3mock to work with the newer stuff. That's covered in https://bugzilla.mozilla.org/show_bug.cgi?id=1517807 .